frontend/package.json

```diff
@@ -4,15 +4,16 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --host",
```

Comment:
Allows the project to be visited in http://<local-IP>:5173/ from all over the localnetwork instead of just localhost in your own computer